src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -33,6 +33,14 @@
                 'Md_Residential',  # Does not map to NLCD
                 'Hd_Residential',  # Does not map to NLCD
                 ],
+    'Imper': np.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 0,          # Impervious surface area percentage # NOQA
```

Comment:
Row 47 of the GMS File Key points me to line 9426 of Class1.vb:

```vb
        ' Default urban landuse lines
        ' - Low Density Mixed
        LuLine(11) = "Ld_Mixed," & LuArea(11).ToString & ",0.15,92,74,60"
        NLoadRate(11) = "0.095,.015,0.33,0"
        PLoadRate(11) = "0.0095,.0021,0.4,0"
        SedLoadRate(11) = "2.8,0.8,0,0"
        ETGrow(11) = 0.85 * (LuArea(11) / BasinAreaHa)
        ETDorm(11) = 0.8 * (LuArea(11) / BasinAreaHa)
```

These are the static values in `LuLine`. Since they are only present in Urban Land uses, I defaulted the rest to 0.